### ISSUE TITLE
External browser doesn't remember file-editing permissions

### ISSUE THREAD

**Issue by Madd0g** [dmarmor/epichrome#327]
Since brave doesn't support the filesystem API, I have to use the external chrome.

I use a website which uses the filesystem API, every time I reopen the app, I need to re-confirm access.
when I go into preferences, under the site permissions settings screen, there are 3 options for file editing: 

- Ask (default)
- Block
- Ask

Other permissions on that screen have "Allow" as one of the options, but not File editing.

Anything I can do to fix this?

**Comment by dmarmor** [comment]
I'm not able to see the behavior you described. I tested with the site:

https://googlechromelabs.github.io/text-editor/

I tried in both Chrome itself, and a Chrome-based app, and it works the same. I'm able to open a file from my file system, edit it, and then save it. The first time I do that, it pops up a dialog asking "Let site edit <file>?" And explains that the site will be able to edit the file until I close all tabs for the site. The only two options are `Cancel` and `Edit file`. If I go to `Chrome settings->Privacy and security->Site Settings->Additional permissions->File editing` it only has a Block list.

Are you certain that Chrome allows sites to save this setting? Is it working differently for you in the actual Chrome browser than it does in your app?
